translator/trace/jaeger/protospan_to_jaegerthrift.go

```diff
@@ -473,3 +479,12 @@ func ocSpanAttributesToJaegerTags(ocAttribs *tracepb.Span_Attributes) []*jaeger.
 
 	return jTags
 }
+
+func handleError(jTag *jaeger.Tag, jTags []*jaeger.Tag) (*jaeger.Tag, []*jaeger.Tag) {
```

Comment:
For consistency this should be follow the naming patern other functions have set (e.g, ocErrorAttributeToJaegerTags) but I think it is simple and localized enough to be placed inline instead of as a function. Can we move this few lines inline?

Reply:
May be we can add a check at the bottom of switch cases before appending? Something like:

```go
if key == error {
   errorTag = &jeagerpb.Tag{key: attrKeyError, vType: jaeger.TagType_BOOL, vValue: true}
    jTag.Key = attrKeyErrorMessage
   jTags = append(jTags, jTag, errorTag) 
}
```

Reply:
Also may be it'd worth it increasing the pre-allocated slice size by one?

Reply:
> For consistency this should be follow the naming patern other functions have set (e.g, ocErrorAttributeToJaegerTags) but I think it is simple and localized enough to be placed inline instead of as a function. Can we move this few lines inline?

 Sure

Reply:
> For consistency this should be follow the naming patern other functions have set (e.g, ocErrorAttributeToJaegerTags) but I think it is simple and localized enough to be placed inline instead of as a function. Can we move this few lines inline?

i will move it inline